nix/haskell-pins.nix

```diff
@@ -59,86 +59,108 @@
         rev = "abf819a4a6ec7601f1e58cb8da13b2fdad377d9e";
         sha256 = "sha256-m1O+F/mOJN5z5WNChmeyHP4dtmLRkl2YnLlTuwzRelk=";
       };
     };
-    cryptobox-haskell = {
-      src = fetchgit {
-        url = "https://github.com/wireapp/cryptobox-haskell";
-        rev = "7546a1a25635ef65183e3d44c1052285e8401608";
-        sha256 = "0dgizj1kc135yzzqdf5l7f5ax0qpvrr8mxvg7s1dbm01cf11aqzn";
-      };
-    };
 
-    # https://github.com/dpwright/HaskellNet-SSL/pull/33
+    # PR: https://github.com/dpwright/HaskellNet-SSL/pull/33
     HaskellNet-SSL = {
       src = fetchgit {
         url = "https://github.com/MangoIV/HaskellNet-SSL";
         rev = "c2844b63a39f458ffbfe62f2ac824017f1f84453";
         sha256 = "sha256-1mu/yEAWr3POY4MHRomum0DDvs5Qty1JvP3v5GS2u64=";
       };
     };
+
     hsaml2 = {
       src = fetchgit {
         url = "https://github.com/wireapp/hsaml2";
-        rev = "51d1fcecebf2417e658b9a78943c84a76a0ed347";
-        sha256 = "sha256-jYJBhXBQ1MTLPI8JsiF2XUtgDxK+eniavNB2B1zaSQg=";
+        rev = "723b377fcd759c8be9ad4b2e159a6a06df0d17c9";
+        sha256 = "sha256-rPfztTu+NR/5FuoYWGMCfJFhrMn4o09bMcEKoerNX4A=";
       };
     };
+
+    # PR: https://github.com/hspec/hspec-wai/pull/49
     hspec-wai = {
       src = fetchgit {
         url = "https://github.com/wireapp/hspec-wai";
-        rev = "6984a06b0c6294677c49d59382d48f975a8733d4";
-        sha256 = "sha256-6FLTMMqvL0xFa5zsMnjVAmdpghmdeBl813bWcOyQo5E=";
+        rev = "08176f07fa893922e2e78dcaf996c33d79d23ce2";
+        sha256 = "sha256-Nc5POjA+mJt7Vi3drczEivGsv9PXeVOCSwp21lLmz58=";
       };
     };
-    saml2-web-sso = {
+
+    # PR: https://gitlab.com/twittner/cql/-/merge_requests/11
+    cql = {
       src = fetchgit {
-        url = "https://github.com/wireapp/saml2-web-sso";
-        rev = "ac46ea888026711860cf784b5bda206873c87333";
-        sha256 = "sha256-IKovI1h2Wkm3Y7Sz6XsxLOv654SgUasaWsDX6gi9hZw=";
+        url = "https://gitlab.com/wireapp/forks/cql";
+        rev = "abbd2739969d17a909800f282d10d42a254c4e3b";
+        sha256 = "sha256-2MYwZKiTdwgjJdLNvECi7gtcIo+3H4z1nYzen5x0lgU=";
       };
     };
-    # MR: https://gitlab.com/twittner/cql-io/-/merge_requests/20
+
+    # PR: https://gitlab.com/twittner/cql-io/-/merge_requests/20
     cql-io = {
       src = fetchgit {
         url = "https://gitlab.com/wireapp/forks/cql-io";
         rev = "c2b6aa995b5817ed7c78c53f72d5aa586ef87c36";
         sha256 = "sha256-DMRWUq4yorG5QFw2ZyF/DWnRjfnzGupx0njTiOyLzPI=";
       };
     };
+
+    # missing upstream PR 
     wai-predicates = {
       src = fetchgit {
-        url = "https://gitlab.com/wireapp/forks/wai-predicates.git";
+        url = "https://gitlab.com/wireapp/forks/wai-predicates";
         rev = "ff95282a982ab45cced70656475eaf2cefaa26ea";
         sha256 = "sha256-x2XSv2+/+DG9FXN8hfUWGNIO7V4iBhlzYz19WWKaLKQ=";
       };
     };
+
+    # we use upstream, but has not been uploaded to hackage since 2016
```

Comment:
Add a comment that this will go away once servantification is complete cc @fisx

Reply:
:+1: